### PR TITLE
Bandchain.js: add getAccount

### PR DESCRIPTION
**Pull request by Pitchanai**
## Implementation details
- Add getAccount

## Please ensure the following requirements are met before submitting a pull request:

- [x] The pull request is targeted against the correct target branch
- [x] The pull request is linked to an issue with appropriate discussion and an accepted design OR is linked to a spec that describes the work.
- [x] The pull request includes a description of the implementation/work done in detail.
- [x] The pull request includes any and all appropriate unit/integration tests
- [ ] You have added a relevant changelog entry to `CHANGELOG_UNRELEASED.md`
- [x] You have re-reviewed the files affected by the pull request (e.g. using the `Files changed` tab in the Github PR explorer)

### CODE REVIEW

File: helpers/bandchain2.js/src/client.ts [hunk 1]
```diff
@@ -49,6 +51,21 @@ export default class Client {
     }
   }
 
+  async getAccount(address: Address): Promise<Account> {
+    const response = await this.getResult(
+      `/auth/accounts/${address.toAccBech32()}`,
+    )
+    console.log(response)
```
Comment (evilpeach) [inline helpers/bandchain2.js/src/client.ts]: remove log

**Review by evilpeach — CHANGES_REQUESTED** [review]

File: helpers/bandchain2.js/src/client.ts [hunk 2]
```diff
@@ -49,6 +51,20 @@ export default class Client {
...
   }
 
+  async getAccount(address: Address): Promise<Account> {
+    const response = await this.getResult(
+      `/auth/accounts/${address.toAccBech32()}`,
+    )
+    const value = response.value
+    return {
+      address: Address.fromAccBech32(value.address),
+      coins: value.coins.map((c: any) => new Coin(c.amount, c.denom)),
```
Comment (evilpeach) [inline helpers/bandchain2.js/src/client.ts]: `c: {amount: number, denom: string}`

File: helpers/bandchain2.js/src/data.ts [hunk 3]
```diff
@@ -16,6 +16,14 @@ export class Coin {
   }
 }
 
+export interface Account {
+  address: Address
+  coins: Coin[]
+  publicKey?: object
+  accountNumber: Number
```
Comment (evilpeach) [inline helpers/bandchain2.js/src/data.ts]: number

**Review by evilpeach — CHANGES_REQUESTED** [review]

File: helpers/bandchain2.js/src/data.ts (hunk 3 above)
Comment (evilpeach) [inline helpers/bandchain2.js/src/data.ts]: number

**Review by evilpeach — APPROVED** [review]

**Review by pzshine — APPROVED** [review]

**Pull request merged**